test/t8_cmesh/t8_gtest_cmesh_partition.cxx

```diff
@@ -48,9 +39,10 @@ class t8_cmesh_partition_class : public testing::TestWithParam<int>{
   void SetUp() override {
     cmesh_id = GetParam();
 
-    if (!(cmesh_id != 89 && (cmesh_id < 237 || cmesh_id > 256))) {
-      GTEST_SKIP();
+    if(cmesh_id == 89 || (cmesh_id>=237 && cmesh_id<=256)){
```

Comment:
```suggestion
    if(cmesh_id == 89 || (237 <= cmesh_id && cmesh_id <= 256)){
```
Just a style-choice. In my opinion this is better, but if you prefer it the other way, I won't stop you from changing it back.